--- a/package/openwrt/net/xl2tpd/files/l2tp.sh
+++ b/package/openwrt/net/xl2tpd/files/l2tp.sh
@@ -14,6 +14,7 @@ proto_l2tp_init_config() {
        proto_config_add_string "keepalive"
        proto_config_add_string "pppd_options"
        proto_config_add_boolean "ipv6"
+       proto_config_add_int "demand"
        proto_config_add_int "mtu"
        proto_config_add_string "server"
        available=1
